docs/platforms-release-process.md

```diff
@@ -169,6 +157,35 @@ Ensure all dependencies and subdependencies have Apache-compatible licenses.
 
     coho check-license -r android
 
+### Get Buy-in
```

Comment:
Why do all the work, and _then_ ask for buy-in?

Reply:
To me "Get Buy-in" meant that I should send the email and wait for another day before continuing with any of the "Before Release" steps. This has led to additional delays since I did not realize there are several more things that I could have checked and fixed while waiting for the response. I will raise another PR to clarify.